Prop/Decidable/_.kind2

```diff
@@ -3,3 +3,8 @@ type Prop.Decidable (p: Type) {
   yep (proof: p)
   nop (proof: Prop.Not p)
 }
+
+// converts a Bool to its "type-theoretic" form
+Prop.Decidable.T (b: Data.Bool) : Type
```

Comment:
Maybe call it somethng like `to_type` and put it in it's own file? Since it's not even in this file.

Maybe even put in Data.Bool